contributors/devel/api-conventions.md

```diff
@@ -595,6 +595,16 @@ Strategic Merge Patch also supports special operations as listed below.
 
 ### List Operations
 
+#### Specifying element ordering for a strategic merge patch on a list [Alpha]
```

Comment:
I think we should specify strategic merge patch elsewhere. This is not the right doc to define it.